provider/manifest/service.go

```diff
@@ -198,7 +208,7 @@ loop:
 				break
 			}
 
-			manager, err := s.ensureManger(req.value.Deployment)
+			manager, err := s.ensureManager(req.value.Deployment)
```

Comment:
omg I never noticed that.  I didn't even follow what you were saying on slack about it.